packages/lib-classifier/src/components/Classifier/components/Toolbar/Toolbar.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Box } from 'grommet'
+import AnnotateButton from './components/AnnotateButton'
+import FieldGuideButton from './components/FieldGuideButton'
+import FullscreenButton from './components/FullscreenButton'
+import MoveButton from './components/MoveButton'
+import ResetButton from './components/ResetButton'
+import RotateButton from './components/RotateButton'
+import ZoomInButton from './components/ZoomInButton'
+import ZoomOutButton from './components/ZoomOutButton'
+
+const toolbarStyles = {
+  border: {
+    color: 'lightGrey',
+    side: 'all'
+  },
+  pad: 'small'
+}
+
+function Toolbar (props) {
+  return (
+    <aside {...props}>
+      <Box {...toolbarStyles}>
+        <AnnotateButton />
+        <MoveButton />
+        <ZoomInButton />
+        <ZoomOutButton />
+        <RotateButton />
+        <FullscreenButton />
+        <ResetButton />
```

Comment:
All of these buttons will need accessible labels, either here or in the individual button components.

Reply:
I put `aria-label`s on the individual button components.

Reply:
👍

Reply:
Can those labels be localised, so that we can handle more than one language.

Reply:
Sure - for now, is it enough to wrap everything in `counterpart` and work out how to collect up the strings later on?

Reply:
Works for me. Saves ourselves from going through the code to pick out English language strings later.

Reply:
Cool, I've added those in.